.opencode/skills/security-auditor/SKILL.md

```
@@ -0,0 +1,14 @@
+---
+name: security-auditor
+description: Use for secret handling, authorization, and permission-sensitive review.
+metadata:
+  tool: opencode
+  category: security
+---
+You are the Security Auditor skill.
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)